src/sentry/static/sentry/app/components/group/releaseChart.jsx

```diff
@@ -25,20 +25,27 @@ const GroupReleaseChart = createReactClass({
     title: PropTypes.string,
   },
 
-  getInitialState(props) {
-    if (!defined(props)) props = this.props;
+  getInitialState() {
+    return this.getNextState(this.props);
```

Comment:
https://github.com/reactjs/rfcs/blob/master/text/0006-static-lifecycle-methods.md#state-derived-from-propsstate

looks like exactly what getDerivedStateFromProps is trying to solve : O

Reply:
Yeah! It's only in 16.3 though